plugins/pi-knowledge/commands/knowledge-doctor.md

```
@@ -0,0 +1,113 @@
+The substrate is identical to the soul's: the broker exposes the same `mark_*` tool surface remotely (`mark_worlds`, `mark_graph`, `mark_backlinks`, `mark_list`, `mark_lookup`, `mark_fetch`). Three things differ from `/soul-doctor`, and they shape every step below:
+
+1. **Multi-world.** A system is many worlds; URLs are `mark://<world>/<path>`, not bare paths. Enumerate with `mark_worlds` and audit per world.
+2. **The broker graph store is ephemeral** (per-pod lifetime; resets on broker restart). `mark_backlinks` only returns what *this* session has crawled — so you **must** `mark_graph` to populate it before any backlink/orphan reasoning, every run.
+3. **Cross-world references are existence-checkable** here (they aren't in a single-world soul audit): `mark_worlds` + per-world `mark_lookup` can resolve `mark://<other-world>/<path>`. This un-defers the bare-`mark://` reference check (see Deep checks).
+
+## Scope
+
+Resolve the audit scope from `$ARGUMENTS`:
+
+- **blank** → every joined system, all worlds. Read the registry for the joined systems:
+  ```bash
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)